crates/forge/Cargo.toml

```diff
@@ -78,6 +78,7 @@ dialoguer = { version = "0.11", default-features = false }
 dunce.workspace = true
 futures.workspace = true
 indicatif = "0.17"
+inferno = "0.11.19"
```

Comment:
It looks like there is an issue with the license of `inferno`, will need to determine if this is compatible with our licenses

https://github.com/jonhoo/inferno/blob/main/LICENSE

Reply:
[cargo-flamegraph](https://github.com/flamegraph-rs/flamegraph) uses it and has the same licenses as us